gbfs.md

```diff
@@ -414,23 +415,23 @@ Field Name | REQUIRED | Type | Defines
 `attribution_organization_name` <br/>*(added in v3.0-RC)* | OPTIONAL | Array&lt;Localized String&gt; | If the feed license requires attribution, name of the organization to which attribution should be provided.
 `attribution_url` <br/>*(added in v3.0-RC)* | OPTIONAL | URL | URL of the organization to which attribution should be provided.
-`brand_assets` <br/>*(added in v2.3)*  | OPTIONAL | Object | An object where each key defines one of the items listed below.
-\- `brand_last_modified` <br/>*(added in v2.3)*  | Conditionally REQUIRED | Date | REQUIRED if `brand_assets` object is defined. Date that indicates the last time any included brand assets were updated or modified.
+`brand_assets` <br/>*(added in v2.3)*  | OPTIONAL | Object |
+\- `brand_last_modified` <br/>*(added in v2.3)*  | REQUIRED | Date | Date that indicates the last time any included brand assets were updated or modified.
 \- `brand_terms_url` <br/>*(added in v2.3)*   | OPTIONAL |  URL |  A fully qualified URL pointing to the location of a page that defines the license terms of brand icons, colors, or other trademark information.  This field MUST NOT take the place of `license_url` or `license_id`.
-\- `brand_image_url` <br/>*(added in v2.3)*  | Conditionally REQUIRED |  URL |  REQUIRED if `brand_assets` object is defined. A fully qualified URL pointing to the location of a graphic file representing the brand for the service. File MUST be in SVG V1.1 format and MUST be either square or round.
+\- `brand_image_url` <br/>*(added in v2.3)*  |  REQUIRED |  URL | A fully qualified URL pointing to the location of a graphic file representing the brand for the service. File MUST be in SVG V1.1 format and MUST be either square or round.
 \- `brand_image_url_dark` <br/>*(added in v2.3)*  | OPTIONAL |  URL | A fully qualified URL pointing to the location of a graphic file representing the brand for the service for use in dark mode applications.  File MUST be in SVG V1.1 format and MUST be either square or round.
 \- `color` <br/>*(added in v2.3)*  | OPTIONAL |  String |  Color used to represent the brand for the service expressed as a 6 digit hexadecimal color code in the form #000000.
 `terms_url` <br/>*(as of v3.0-RC)* | OPTIONAL | Array&lt;Localized String&gt; | A fully qualified URL pointing to the terms of service (also often called "terms of use" or "terms and conditions") for the service.
-`terms_last_updated` <br/>*(added in v2.3)* |Conditionally REQUIRED | Date | REQUIRED if `terms_url` is defined. The date that the terms of service provided at `terms_url` were last updated. 
+`terms_last_updated` <br/>*(added in v2.3)* | REQUIRED | Date | The date that the terms of service provided at `terms_url` were last updated. 
```

Comment:
I like that the proposed solution is more compact and less verbose. I am concerned that the proposed solution may be less obvious for the newcomers to the spec.

Curious to know what @josee-sabourin and others think.

Reply:
I have the opposite feeling : conditionals are hard to read, and having a required field in an optional object is, in my opinion, easier to understand, especially for newcomers. But I want to see if others are feeling something else.

Reply:
Tagging recent contributors to get your opinion: @mplsmitch, @testower, @cmonagle, @simonsolnes, @ezmckinn, @AntoineAugusti, @ArashMansouri, @jkurzanski.

What is your view on the proposal to make the specification less verbose for REQUIRED fields inside OPTIONAL objects?

Before | After
-- | --
Conditionally REQUIRED with the mention "REQUIRED if object is defined" | REQUIRED

Reply:
I agree with @tdelmas. If the object itself is optional, there's no need to repeat that information on each property of the object.

Reply:
+1 for the changes!